### PR TITLE
TXAD-12: Implement Report New Incident Screens.

### PR DESCRIPTION
**Pull request by ruchawajpe**

### CODE REVIEW

File: lib/screens/incident/widgets/incident_select_health_and_safety_expansion_tile.dart [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  Widget build(BuildContext context) {
+    context.read<ReportNewIncidentBloc>().add(
+        ReportNewIncidentAnonymousExpansionChange(
+            reportIncidentAnonymousId: ''));
+    return BlocBuilder<ReportNewIncidentBloc, ReportNewIncidentStates>(
+        buildWhen: (previousState, currentState) =>
+            currentState is ReportNewIncidentAnonymousSelected,
+        builder: (context, state) {
+          if (state is ReportNewIncidentAnonymousSelected) {
+            // addIncidentMap['identity'] = state.reportAnonymousId;
```
Comment (Aditi-005) [inline lib/screens/incident/widgets/incident_select_health_and_safety_expansion_tile.dart:29]: Remove commented code

File: lib/screens/incident/widgets/incident_site_list.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const IncidentSiteList(
+      {Key? key,
+      required this.fetchIncidentMasterModel,
+      required this.selectSiteName})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: const GenericAppBar(title: 'Select Site'),
```
Comment (Aditi-005) [inline lib/screens/incident/widgets/incident_site_list.dart:23]: Add in string constant or get from database

File: lib/utils/incident_select_category_util.dart [hunk 3]
```diff
@@ -0,0 +1,56 @@
+// import '../data/models/incident/fetch_incident_master_model.dart';
```
Comment (Aditi-005) [inline lib/utils/incident_select_category_util.dart:1]: Remove file

**Review by Aditi-005 — CHANGES_REQUESTED** [review]

File: lib/screens/incident/widgets/incident_select_health_and_safety_expansion_tile.dart (hunk 1 above)
Comment (ruchawajpe) [reply lib/screens/incident/widgets/incident_select_health_and_safety_expansion_tile.dart:29]: Done

File: lib/screens/incident/widgets/incident_site_list.dart (hunk 2 above)
Comment (ruchawajpe) [reply lib/screens/incident/widgets/incident_site_list.dart:23]: Done

File: lib/utils/incident_select_category_util.dart (hunk 3 above)
Comment (ruchawajpe) [reply lib/utils/incident_select_category_util.dart:1]: Done

**Review by Aditi-005 — CHANGES_REQUESTED** [review]
Please make changes

**Review by Aditi-005 — APPROVED** [review]

**Pull request merged**